src/diagnostic-channel-publishers/package.json

```diff
@@ -24,8 +25,8 @@
     "@types/pg": "^7.4.11",
     "diagnostic-channel": "0.2.0",
     "mocha": "^3.2.0",
-    "mongodb": "^2.2.25",
-    "mysql": "~2.14.1",
+    "mongodb": "^3.1.13",
```

Comment:
We need to continue testing mongodb 2.2 as well

Reply:
added 2.x testing, as well as 3.0.5 testing